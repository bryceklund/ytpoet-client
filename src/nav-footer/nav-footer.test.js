import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'
import HeadFoot from './nav-footer'



describe(`Nav+Footer`, () => {
    const { Nav, Footer } = HeadFoot
    it(`renders without crashing`, () => {
        const div = document.createElement('div')
        ReactDOM.render(<BrowserRouter>
                            <Nav />
                            <Footer />
                        </BrowserRouter>, div)
        ReactDOM.unmountComponentAtNode(div)
    })
    it(`matches its snapshot`, () => {
        const tree = renderer.create(<BrowserRouter>
                                        <Nav />
                                        <Footer />
                                    </BrowserRouter>)
                                    .toJSON()
        expect(tree).toMatchSnapshot()
    })
})