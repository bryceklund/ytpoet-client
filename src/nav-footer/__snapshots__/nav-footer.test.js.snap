// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav+Footer matches its snapshot 1`] = `
Array [
  <nav
    className="nav"
  >
    <h1>
      <a
        className="hero-link"
        href="/home"
        onClick={[Function]}
      >
        ytPoet
      </a>
    </h1>
  </nav>,
  <footer>
    Â© 2020 bryce eklund | 
    <a
      href="https://github.com/bryceklund"
    >
      github
    </a>
  </footer>,
]
`;
